### ISSUE TITLE
This error means that you don't have an Office 365 plan that includes Office 365 ProPlus. Therefore, you can't use shared computer activation.

### ISSUE THREAD

**Issue by StuBeck32** [MicrosoftDocs/OfficeDocs-DeployOffice#380]
Well what licensing can use this feature? This is the first response on a search and I shouldn't have to click another link to find out the problem. Tell 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e327cac6-ceac-fa8b-18ce-7312023a4acb
* Version Independent ID: b8006cfe-a158-0ef4-075c-089329094333
* Content: [Troubleshoot issues with shared computer activation for Office 365 ProPlus](https://docs.microsoft.com/en-us/deployoffice/troubleshoot-issues-with-shared-computer-activation-for-office-365-proplus#feedback)
* Content Source: [DeployOffice/troubleshoot-issues-with-shared-computer-activation-for-office-365-proplus.md](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/blob/live/DeployOffice/troubleshoot-issues-with-shared-computer-activation-for-office-365-proplus.md)
* Service: **o365-proplus-itpro**
* GitHub Login: @DHB-MSFT
* Microsoft Alias: **danbrown**

**Comment by tcrobbins4** [comment]
So, Microsoft, WHAT PLAN do we need to purchase.  I've tried about 3 different plans so far and each of them says it doesn't support shared activation.  I imagine I have to buy the most expensive plan even though cheaper plans include Office.  SMH...

**Comment by Shachar2like** [comment]
ok fine I have a license for one computer and one user profile, why isn't a solution listed?
should I go into another profile and do what? unactivate office? is that possible?

**Comment by DHB-MSFT** [comment]
@djbrakke -- Which subscription plan do you have? Shared computer activation is available if you have the **_Microsoft 365 Business_** plan, but isn't available if you have the Office 365 Business or Office 365 Business Premium plan.  

And shared computer activation is available for any plan that includes Office 365 ProPlus, such as Office 365 E3/E5 or Microsoft 365 E3/E5.

**Comment by StuBeck32** [comment]
Where else is this documented Daniel? We've been able to setup shared activation with a business premium account. If its the case where the specific version of ProPlus included with O365 Biz or Biz Premium, that you rename the version of office used there. "If you have an Office 365 plan that includes Office 365 ProPlus, you can use shared computer activation" is clearly not correct if what you say is the case.

**Comment by StuBeck32** [comment]
Maybe that explains all this confusion. I rebuilt my PC on Thursday and downloaded Office, now I have Office 365 Business. But everyone else in the company has the same plan as me, but has Office 365 ProPlus. We all have the same Office 365 Business Premium plan. I'm assuming there was an issue on Microsoft's end when Access was added where we were incorrectly given ProPlus when we should have been given the updated version of Premium. 

When I created this note in June, the documentation at the time definitely indicated that Office 365 Business Premium included ProPlus. Now it looks like that error may have been resolved.
